### DISCUSSION TITLE
Feature: Add a way to upload two files with the same name without overwriting it

### DISCUSSION THREAD

**Discussion by Felitendo** [diced/zipline#481] [Ideas]
So basically when you upload a file named "download.png" it creates the link "https://files.example.com/u/download.png". When you upload another file wich is different to the first one, but it's name is also "download.png" then it will upload to "https://files.example.com/u/download.png" and therefore overwriting the already existing one. It would be really cool if you can set it up, that zipline automatically renames it to "https://files.example.com/download-2.png".

**Comment by diced** [comment]
I don't think I will have a system that automatically renames files, but as of the latest release (v3.7.5) it should error out when you upload the same file.

**Reply by Felitendo** [reply]
It showes an error only when you upload the file on the website. But if you upload it via sharex, it doesn't show anything. Btw how is V4 doing?

**Reply by diced** [reply]
How is that possible when they use the same endpoint?

Also, you can check the status and progress of v4 on the roadmap.

**Comment by diced** [comment]
Closing as not planned & overwriting issue was fixed in the latest releases
